docs/lib/datastore/fragments/ios/getting-started/30_platformIntegration.md

```diff
@@ -8,17 +8,24 @@ pod 'Amplify/Tools'
 
 Then **add a Build Phase** to your app target.
 
-1. Click on your project in the file list, choose your target under TARGETS, click the Build Phases tab.
-2. Add a **New Run Script Phase** by clicking the little plus icon in the top left.
-3. Drag the new Run Script phase **above the Compile Sources phase**, expand it and paste the following script:
-    ```console
-    "${PODS_ROOT}/AmplifyTools/amplify-tools.sh"
-    ```
-4. Build the project (`Cmd+b`)
+1. Click on the **Todo project** in the project workspace, then click the **Todo** app target, and then **click on `Build Phases`**.
+2. Tap the `+` button to add another phase, and **select "New Run Script Phase"**
+3. Drag the new `Run Script` phase to move the phase so that it runs prior to the `Compile Sources` phase.
+4. Update the `Run Script` build phase title to **"Run Amplify Tools"**, and then update the shell script to have a single line with:
+  ```bash
+  ${PODS_ROOT}/AmplifyTools/amplify-tools.sh
```

Comment:
This needs to have quotes around it like:
"${PODS_ROOT}/AmplifyTools/amplify-tools.sh"
to support projects that may have a space around it.

Reply:
i'm guessing we can handle this later?

Reply:
it's addressed in part 2. Sorry, I made the change in the other branch without realizing it.